src/listeners/messageReactionAdd.ts

```diff
@@ -24,7 +24,9 @@ client.on("messageReactionAdd", async (reaction, user) => {
 		/https:\/\/crowdin\.com\/translate\/\w+\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$_.+!*'()-]*)?#\d+/gi.test(reaction.message.content!)
 	) {
 		const language = await db.collection<MongoLanguage>("languages").findOne({ code: channel.name.split("-")[0] }),
-			role = channel.guild!.roles.cache.find(r => r.name === `${language!.name} Proofreader`)
+			role = channel.guild!.roles.cache.find(r => r.name === `${language!.name} Proofreader`),
+			reactedToOwn = reaction.message.author.bot ? reaction.message.content.includes(user.id) : reaction.message.author.id === user.id
```

Comment:
To avoid all the ifs repeating down below you can just make a single if reactedToOwn that removes the reaction and stops the listener below this